### DISCUSSION TITLE
Can I use the repo to concert HTML table to PNG file?

### DISCUSSION THREAD

**Discussion by zydjohnHotmail** [SixLabors/ImageSharp#1967] [Ideas]
Hello:
I have a simple HTML table, like this:
<table style="width:100%">
  <tr>
    <th>#</th>
    <th>#</th>
    <th>#</th>
    <th>#</th>
  </tr>
  <tr>
    <td style="border: 3px solid red; ">1</td>
    <td style="border: 3px solid red; ">1</td>
    <td style="border: 3px solid red; ">1</td>
    <td style="border: 3px solid red; ">1</td>
  </tr>
  <tr>
    <td>2</td>
    <td>2</td>
    <td>2</td>
    <td>2</td>
  </tr>
  <tr>
    <td>3</td>
    <td>3</td>
    <td>3</td>
    <td>3</td>
  </tr>
</table>
Can I use the repo to convert the HTML table to PNG file?

**Comment by tocsoft** [comment]
nope... this is an imaging library you are looking for a html renderer.

**Comment by JimBobSquarePants** [comment]
@zydjohnHotmail Please note this is a misuse of our discussions forum. This channel is for feature ideas/requests only.
